test_utils.go

```diff
@@ -561,3 +570,31 @@ func checkAPIAccess(t *testing.T, client HTTPClient, stream string, role string)
 		require.Equalf(t, 403, response.StatusCode, "Server returned http code: %s and response: %s", response.Status, readAsString(response.Body))
 	}
 }
+
+func activateHotTier(t *testing.T) {
+	payload := StreamHotTier{
+		Size: fmt.Sprintf("%d", int64(20*math.Pow(1024, 3))), // set hot tier size to be 20 GB
+	}
+	json, _ := json.Marshal(payload)
+
+	req, _ := NewGlob.QueryClient.NewRequest("PUT", "logstream/"+NewGlob.Stream+"/hottier", bytes.NewBuffer(json))
+	req.Header.Set("Content-Type", "application/json")
+	response, err := NewGlob.QueryClient.Do(req)
+	body := readAsString(response.Body)
+
+	if NewGlob.IngestorUrl.String() != "" {
+		require.Equalf(t, 200, response.StatusCode, "Server returned unexpected http code: %s and response: %s", response.Status, body)
+		require.NoErrorf(t, err, "Activating hot tier failed in distributed mode: %s", err)
+	} else {
+		// right now, hot tier is unavailable in standalone so anything other than 200 is fine
+		require.NotEqualf(t, 200, response.StatusCode, "Hot tier has been activated in standalone mode: %s and response: %s", response.Status, body)
+	}
+}
+
+func disableHotTier(t *testing.T) {
+	req, _ := NewGlob.QueryClient.NewRequest("PUT", "logstream/"+NewGlob.Stream+"/hottier", nil)
```

Comment:
@vishalkrishnads there is no api for disable hot tier, rather use -
DELETE /logstream/{logstream}/hottier

Reply:
sorry didn't notice. changed now.